### PR TITLE
Exercises from chapter3

### PR DESCRIPTION
**Pull request by liliamyhaliuk**
I did all exercises from chapter 3 of the book "Think Python". Please, check when you have time.
I'm not sure about the 3 exercise. It looks like my decision is too big but using concatenation is forbidden by the definition of this exercise...

### CODE REVIEW

File: chapter3/3.14.2.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+def print_spam(s):
+    print(s)
+
+def do_four(func, val):
+    '''Run a function four times with an argument val'''
+    do_twice(func, val)
+    do_twice(func, val)
+
+do_four(print_spam, 'spam')
```
Comment (auraz) [inline chapter3/3.14.2.py:26]: Correct

File: chapter3/3.14.3.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    '''Draw a graph with 2 rows and 2 columns'''
+    make_line(draw_plus_dash, '+')
+    do_four(make_line, draw_slash_blank, '|')
+    make_line(draw_plus_dash, '+')
+    do_four(make_line, draw_slash_blank, '|')
+    make_line(draw_plus_dash, '+')
+
+draw_graph()
+
+# 2. Write a function that draws a similar grid with four rows and four columns.
```
Comment (auraz) [inline chapter3/3.14.3.py:56]: only 2 rows for me.

**Review by auraz — CHANGES_REQUESTED** [review]
Good! Please address comments.

File: chapter3/3.14.1.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+'''
+Exercises of the book "Think python"
+3.14.1 Exercises:
+'''
+# Write a function named right_justify that takes a string named s as a parameter
+# and prints the string with enough leading spaces
+# so that the last letter of the string is in column 70 of the display.
+
+def right_justify(string):
+    '''Move argument to the left so the last letter of the argument is in column 70 of display'''
```
[the remark this answers is not in the document]
Comment (liliamyhaliuk) [reply chapter3/3.14.1.py]: Hi!
I updated the docstring in all functions with the right quotation mark.

File: chapter3/3.14.1.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    left_space = 70 - len(string)
+    print(left_space*' ' + string)
+
+right_justify('s')
```
[the remark this answers is not in the document]
Comment (liliamyhaliuk) [reply chapter3/3.14.1.py:14]: Thank you for checking!

File: chapter3/3.14.2.py (hunk 1 above)
Comment (liliamyhaliuk) [reply chapter3/3.14.2.py:26]: Thank you!

**Review by auraz — APPROVED** [review]
Much better!

**Pull request merged**